treenode.go

```diff
@@ -679,6 +681,9 @@ func (n *TreeNodeInstance) SetConfig(c *GenericConfig) {
 	n.configMut.Lock()
 	defer n.configMut.Unlock()
 	n.config = c
+	for id := range n.sentTo {
+		n.sentTo[id] = false
```

Comment:
does `SetConfig` get called only once? If not one needs to make sure to clean the map so the new config gets send to everyone. Otherwise I agree that it is not needed